code/game/machinery/camera/camera.dm

```diff
@@ -101,8 +101,7 @@ var/list/camera_names=list()
 	var/nethash=english_list(network)
 	camera_names[nethash+c_tag]=null
 	..()
-	if(name != replacetext(name,oldarea.name,newarea.name))
-		name_camera()
+	name_camera()
```

Comment:
It actually didn't work before I removed it.  All security cameras just get named "security camera" so their c_tag never updated.  :+1: :+1: :+1: